diff --git a/nemo_rl/models/policy/workers/patches.py b/nemo_rl/models/policy/workers/patches.py
index 7c78c8a..dbcd423 100644
--- a/nemo_rl/models/policy/workers/patches.py
+++ b/nemo_rl/models/policy/workers/patches.py
@@ -117,8 +117,8 @@ def apply_torch_aten_alias_tensor_patch():
     in PyTorch 2.9. See https://github.com/pytorch/pytorch/pull/166867 for the upstream fix.
     We can remove this patch when we upgrade torch to include this fix.
     """
-    assert torch.__version__.startswith("2.9.0"), (
-        "This patch is needed for torch 2.9.0. Please retest if you upgrade torch to a newer version and remove this patch."
+    assert torch.__version__.startswith("2.9."), (
+        "This patch is needed for torch 2.9.x. Please retest if you upgrade torch to a newer version and remove this patch."
     )
     try:
         register_op_strategy(torch.ops.aten.alias.default)(
